src/services/editor/helper/fakeParams.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * This fake passing die user id to params, that the editor MS has a valid input.
```

Comment:
you mean "the user id", I suppose 😉 I don't understand what the function does from this comment, so it only confuses me...